packages/firestore/test/unit/local/persistence_test_helpers.ts

```diff
@@ -44,3 +55,42 @@ export async function testMemoryPersistence(): Promise<MemoryPersistence> {
   await persistence.start();
   return persistence;
 }
+
+/**
+ * Creates LocalStorageNotificationChannel instance for testing, destroying any
+ * previous contents if they existed.
+ */
+export async function testLocalStorageNotificationChannel(
+  ownerId: string,
+  existingMutationBatches: BatchId[],
+  existingQueryTargets: TargetId[]
+): Promise<TabNotificationChannel> {
+  window.localStorage.clear();
```

Comment:
I wonder if this is dangerous to do...  You could imagine the test runner using local storage or something...  Consider iterating localStorage and only deleting things with the firestore prefix...  just in case?

Reply:
Hm. Hrmm. Okay.